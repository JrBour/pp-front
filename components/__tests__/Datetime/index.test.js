import { shallowMount } from '@vue/test-utils'
import Datetime from '@/components/Datetime'

describe('Datetime', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallowMount(Datetime, {
      propsData: {
        text: 'Lorem ipsum',
        id: 'id',
        name: 'name',
        error: '',
        onChange: jest.fn(),
      },
    })
  })

  describe('snapshot', () => {
    it('renders properly', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
