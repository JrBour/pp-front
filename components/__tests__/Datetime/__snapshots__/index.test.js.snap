// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Datetime snapshot renders properly 1`] = `
<div class="input__wrapper">
  <div class="input__container"><input id="id" name="name" type="text" class=""> <label for="id" class="">
      Lorem ipsum <span>*</span></label></div>
  <p></p>
</div>
`;
